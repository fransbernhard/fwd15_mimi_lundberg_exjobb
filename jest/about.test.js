/**
* @jest-environment jsdom
*/

import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import About from '../src/components/about';

describe('components', () => {
  describe('<About/>', () => {
    it('renders correctly', () => {
      const tree = renderer
        .create(
          <About />
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
