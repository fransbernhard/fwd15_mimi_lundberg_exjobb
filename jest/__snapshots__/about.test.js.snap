// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <About/> renders correctly 1`] = `
<div
  className="about"
  id="about"
>
  <div
    className="wrapper-about"
  >
    <div
      className="wrapper-a"
    >
      <img
        className="mySlides about-img"
        src="magden.jpg"
      />
      <p>
        Magda har massa csn lån och nästan en kandidat  
        <br />
        i företagsekonomi. För daglig uppdatering --&gt;
        <br />
         
        <a
          href="https://www.instagram.com/magdenmagden/?hl=sv"
          target="_blank"
        >
          @magdenmagden
        </a>
      </p>
      <br />
    </div>
    <div
      className="wrapper-cv"
    >
      <h5>
        2017
      </h5>
      <p>
        Vernisage @ Layered Official | Stockholm 
        <br />
        Försäljning genom Young Art | Stockholm 
        <br />
        Utställning Nividas/Adornment | Stockholm
      </p>
      <br />
      <h5>
        2016
      </h5>
      <p>
        Vernissage DÅLIG SIKT | Stockholm 
        <br />
        Utställning | L´homme rouge, Stockholm
        <br />
        Artwork "Dying for you" | Otto Knows
      </p>
      <br />
      <h5>
        2015
      </h5>
      <p>
        Vernissage STRÖSSEL | Stockholm
        <br />
        Utställning | Kaffebaren, Malmö
        <br />
        Utställning | Scandic hotel, Malmö
        <br />
        Artwork "Next to me" | Otto Knows
      </p>
      <br />
      <h5>
        2014
      </h5>
      <p>
        Vernissage BOKSLUT | Stockholm
        <br />
        Utställning | Brillo, Stockholm
        <br />
        Utställning | Mood gallerian, Stockholm
        <br />
        Utställning | Höganäs konsthall, Höganäs
      </p>
      <br />
      <h5>
        2013
      </h5>
      <p>
        Vernissage TORSK | Stockholm
        <br />
        Utställning | East, Stockholm
      </p>
      <br />
      <p>
        2011-2012 Lunds Konst och Designskola | Lund
        <br />
        2012-2013 Beckmans kvällsskola | Stockholm
      </p>
    </div>
  </div>
</div>
`;
